### DISCUSSION TITLE
[Bug] CROSS_PROCESS

### DISCUSSION THREAD

**Discussion by cunyouchen** [apache/skywalking#11440] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

<img width="979" alt="image" src="https://github.com/apache/skywalking/assets/62415257/9f779c84-7adc-4c7e-a9d6-f9c81015c99d">

This error should be caused by a plugin bug. Is there a configuration that can solve this problem, or when will the official fix this plugin issue

### What you expected to happen

Someone has raised this issue, but there is no final solution

### How to reproduce

This problem has caused our production accidents

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
We have a FAQ about this, https://skywalking.apache.org/docs/main/next/en/faq/vnode/

And the simple answer is, there is impossible to have this kind of configuration. This is a visualization of a data lost statement.
